android/build.gradle

```diff
@@ -22,6 +22,7 @@ rootProject.allprojects {
 apply plugin: 'com.android.library'
 
 android {
+    namespace "flutter.plugins.vibrate"
```

Comment:
looks like:
```gradle
    // Conditional for compatibility with AGP <4.2.
    if (project.android.hasProperty("namespace")) {
      namespace "flutter.plugins.vibrate"
    }
```
would be the way other packages (like, patrol, sentry_flutter, cloud_firestore, mobile_scanner) go